secure_fw/CMakeLists.txt

```
@@ -47,10 +47,18 @@ target_compile_options(tfm_s
         ${COMPILER_CP_FLAG}
 )
 
+if(${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    # To get debug symbols of function called through veneer functions
+    file(STRINGS ${CMAKE_BINARY_DIR}/generated/tools/tfm_veneers_symbols.txt VENEER_SYMBOLS)
```

```suggestion
    file(STRINGS ${CMAKE_BINARY_DIR}/generated/tools/tfm_veneers_symbols.txt veneer_symbols)
```

lower-case is (generally, not sure about TF-M) used for locally scoped variables like VENEER_SYMBOOLS.